CHANGELOG.md

```diff
@@ -2,9 +2,41 @@
 
 ## Next (unreleased)
```

Comment:
I don't think you need this `Next (unreleased)` line?

Reply:
:+1: